app/experimenter/nimbus-ui/src/lib/mocks.tsx

```diff
@@ -401,3 +405,86 @@ export const mockGetStatus = (status: NimbusExperimentStatus | null) => {
   const { experiment } = mockExperimentQuery("boo", { status });
   return getStatus(experiment);
 };
+
+const fiveDaysAgo = new Date();
+fiveDaysAgo.setDate(fiveDaysAgo.getDate() - 5);
+
+/** Creates a single mock experiment suitable for getAllExperiments queries.  */
+export function mockSingleDirectoryExperimentFactory(
```

Comment:
We're not including the word `Factory` in similarly named helpers, could remove for brevity, but also fine with leaving.